sdk/resourcemanager/Azure.ResourceManager.Core/src/Custom/Models/Location.cs

```diff
@@ -12,212 +12,212 @@ namespace Azure.ResourceManager.Core
     /// <summary>
     /// Represents an Azure geography region where supported resource providers live.
     /// </summary>
-    public class LocationData : IEquatable<LocationData>, IComparable<LocationData>
+    public partial class Location : IEquatable<Location>, IComparable<Location>
     {
         #region Public Cloud Locations
 
         /// <summary>
         /// Public cloud location for East Asia.
         /// </summary>
-        public static readonly LocationData EastAsia = new LocationData { Name = "eastasia", CanonicalName = "east-asia", DisplayName = "East Asia" };
+        public static readonly Location EastAsia = new Location { Name = "eastasia", CanonicalName = "east-asia", DisplayName = "East Asia", RegionalDisplayName = "(Asia Pacific) East Asia" };
```

Comment:
Haven't thought this through, but is there an argument for these being ref readonly?